pages/executive/[proposal-id].tsx

```diff
@@ -301,7 +301,45 @@ const ProposalView = ({ proposal }: Props): JSX.Element => {
                   scrollbarWidth: 'none'
                 }}
               >
-                {supporters ? (
+                {!allSupporters && !supportersError && (
+                  <Flex
+                    sx={{
+                      height: '100%',
+                      justifyContent: 'center',
+                      alignItems: 'center'
+                    }}
+                  >
+                    <Spinner size={32} />
+                  </Flex>
+                )}
+
+                {supportersError && (
+                  <Flex
+                    sx={{
+                      height: '100%',
+                      justifyContent: 'center',
+                      alignItems: 'center',
+                      fontSize: 4,
+                      color: 'onSecondary'
+                    }}
+                  >
+                    List of supporters currently unavailable
+                  </Flex>
+                )}
+                {allSupporters && (!supporters || supporters.length === 0) && (
+                  <Flex
+                    sx={{
+                      height: '100%',
+                      justifyContent: 'center',
+                      alignItems: 'center'
+                    }}
+                  >
+                    <Text>Currently there &apos;t any supporters</Text>
```

Comment:
will this resolve correctly? Looks like `aren` is missing? Maybe just say "Currently there are no supporters" :P

Reply:
Nice catch. Agree with using `Currently there are no supporters` instead.